modules/fieldlinessequence/rendering/renderablefieldlinessequence.cpp

```diff
@@ -40,324 +40,467 @@
     };
     constexpr openspace::properties::Property::PropertyInfo MaskingQuantityInfo = {
-        "maskingQuantity",
+        "MaskingQuantity",
         "Quantity used for Masking",
         "Quantity used for masking."
     };
-    constexpr openspace::properties::Property::PropertyInfo OriginButtonInfo = {
-        "focusCameraOnParent",
-        "Focus Camera",
-        "Focus camera on parent."
+    constexpr openspace::properties::Property::PropertyInfo LineWidthInfo = {
+        "LineWidth",
+        "Line Width",
+        "This value specifies the line width of the field lines if the "
+        "selected rendering method includes lines."
     };
     constexpr openspace::properties::Property::PropertyInfo TimeJumpButtonInfo = {
-        "timeJumpToStart",
+        "TimeJumpToStart",
         "Jump to Start Of Sequence",
         "Performs a time jump to the start of the sequence."
     };
 
-    enum class SourceFileType : int {
-        Cdf = 0,
-        Json,
-        Osfls,
-        Invalid
-    };
+    struct [[codegen::Dictionary(RenderableFieldlinesSequence)]] Parameters {
+        enum class SourceFileType {
+            Cdf,
+            Json,
+            Osfls
+        };
+        // Input file type. Should be cdf, json or osfls
+        SourceFileType inputFileType;
 
-    float stringToFloat(const std::string& input, float backupValue = 0.f) {
-        float tmp;
-        try {
-            tmp = std::stof(input);
-        }
-        catch (const std::invalid_argument& ia) {
-            LWARNING(fmt::format(
-                "Invalid argument: {}. '{}' is NOT a valid number", ia.what(), input
-            ));
-            return backupValue;
-        }
-        return tmp;
-    }
+        // Should be path to folder containing the input files
+        std::filesystem::path sourceFolder [[codegen::directory()]];
+
+        // Path to a .txt file containing seed points. Mandatory if CDF as input.
+        // Files need time stamp in file name like so: yyyymmdd_hhmmss.txt
+        std::optional<std::filesystem::path> seedPointDirectory [[codegen::directory()]];
+
+        // Currently supports: batsrus, enlil & pfss
+        std::optional<std::string> simulationModel;
+
+        //sim mod
+        //std::optional<openspace::fls::Model> model;
+
+        // Extra variables such as rho, p or t
+        std::optional<std::vector<std::string>> extraVariables;
+        
+        // Which variable in CDF file to trace. b is default for fieldline
+        std::optional<std::string> tracingVariable;
+
+        // 1.f is default, assuming meters as input.
+        // Convert the models distance unit, ex. AU for Enlil, to meters.
+        // Can be used during runtime to scale domain limits.
+        std::optional<float> scaleToMeters;
+
+        // If False (default) => Load in initializing step and store in RAM
+        std::optional<bool> loadAtRuntime;
+
+        // Values should be paths to .txt files
```

Comment:
containing what? Since this is the documentation describing to our users what the parameters are useful, they deserve some more elaborate descriptions IMO. This goes for all the parameters above as well. 

Maybe go through them all and write some more easy-to-understand sentences? For example, in this case: 

"A list of paths to .txt files containing the color tables to be used for..."